### PR TITLE
Skip test_flow_build_executable

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by wangchao1230 — DISMISSED** [review]

File: src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py [hunk 1]
```diff
@@ -1121,6 +1121,7 @@ def get_node_settings(_flow_dag_path: Path):
             )
             assert get_node_settings(Path(source)) != get_node_settings(new_flow_dag_path)
 
+    @pytest.mark.skip("Raise Exception: Process terminated with exit code 4294967295 in windows CI")
```
Comment (brynn-code) [inline src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py:1124]: use @pytest.mark.skipif(sys.platform == "win32", reason="...") to filter platform?

Comment (YingChen1996) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py]: updated

File: src/promptflow/promptflow/_sdk/operations/_flow_operations.py [hunk 2]
```diff
@@ -377,7 +377,7 @@ def _build_as_executable(
             import streamlit_quill  # noqa: F401
         except ImportError as ex:
             raise UserErrorException(
-                f"Please install PyInstaller, streamlit and streamlit_quill for building " f"executable, {ex.msg}. "
+                f"Please install PyInstaller, bs4, streamlit and streamlit_quill for building executable, {ex.msg}. "
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/operations/_flow_operations.py:380]: I think we should not print the package name in the error message, as long as user install promptflow[executable]

File: src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py (hunk 1 above)
Comment (wangchao1230) [inline src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py]: can we only skip in windows?

**Review by wangchao1230 — APPROVED** [review]

File: src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py (hunk 1 above)
Comment (YingChen1996) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py]: have updated, only skip win32

**Review by brynn-code — DISMISSED** [review]

File: src/promptflow/promptflow/_sdk/operations/_flow_operations.py (hunk 2 above)
Comment (YingChen1996) [reply src/promptflow/promptflow/_sdk/operations/_flow_operations.py]: updated

**Review by brynn-code — APPROVED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Review by brynn-code — APPROVED** [review]

**Pull request merged**